exchange/howto/manual-dispute-payout.adoc

```diff
@@ -0,0 +1,184 @@
+
+Send the trader their public key (as determined in the steps above) and a link to the [trader instructions] section of this document below. Ask them to follow those instructions and to respond to you with their private key value once they have it.
+
+When you have the trader's private key, move on to the next step.
+
+=== Step 4. Issue the manual dispute payout
+
+Now that you have all the necessary information, you can *open the Emergency multi-sig payout tool* by clicking `alt-g` in the dispute view:
+
+image:images/multisig-payout-tool.png[Emergency multi-sig payout tool]
+
+NOTE: Several values are pre-filled in the payout tool. They are a left-over from testing in an earlier version of Bisq. *You should erase these pre-filled values and replace them with the ones you have collected in the steps above*.
+
+depositTxHex:: The Deposit transaction ID from the Dispute details view, i.e. the same transaction ID you clicked on to view the transaction in a block explorer.
+
+buyerPayoutAmount:: The amount in BTC that the buyer should be paid out, as noted above, e.g. `0.03`
+
+sellerPayoutAmount:: The amount in BTC the seller should be paid out, e.g. `0.0662`
+
+arbitratorPayoutAmount:: This value should always be `0`, as we no longer issue payouts to arbitrators
+
+Tx fee:: The BTC value of the `txFee` in Satoshis as taken from the JSON contract details above, e.g. `0.00036`
+
+buyerAddressString:: The buyer's bitcoin address
+
+sellerAddressString:: The seller's bitcoin address
+
+arbitratorAddressString:: The arbitrator's bitcoin address
+
+buyerPrivateKeyAsHex:: The buyer's private key (leave empty if buyer was not the winner)
+
+sellerPrivateKeyAsHex:: The seller's private key (leave empty if seller was not the winner)
+
+arbitratorPrivateKeyAsHex:: The arbitrator's private key
+
+buyerPubKeyAsHex:: The buyer's public key (leave empty if buyer was not the winner)
+
+sellerPubKeyAsHex:: The seller's public key (leave empty if buyer was not the winner)
+
+arbitratorPubKeyAsHex:: The arbitrator's public key
+
+P2SHMultiSigOutputScript:: The P2SH multi-sig output script hex value as copied from the block explorer
+
+When all values have been entered (and double-checked!), click `Sign and publish transaction` to issue the payout.
+
+When you have confirmed that the payout transaction worked as expected, *let the trader know and close out the ticket*.
+
+
+== Trader instructions
+
+[WARNING]
+The instructions below require you to share sensitive private key information with your arbitrator in order to assist in a manual multi-sig payout. This is necessary only in extraordinary situations, e.g. when a bug or network issue has caused the normal, automated multi-sig payout process to fail. In the unlikely event that you do need to follow these instructions, *you should never share private key information with anyone other than your arbitrator*, and *you should send your private key information only via the Bisq support ticket chat interface*.
```

Comment:
Suggested addition:
The private key the arbitrator requests from you is only used for payout from the  MultiSig address of that particular trade. All other trades have different addresses and keys. For Bitcoins you are holding in your Bisq wallet other addresses and keys are used as well.